app/uk/gov/hmrc/claimvatenrolmentfrontend/forms/CaptureLastMonthSubmittedForm.scala

```diff
@@ -16,15 +16,27 @@
 
 package uk.gov.hmrc.claimvatenrolmentfrontend.forms
 
-import play.api.data.Form
-import play.api.data.Forms.boolean
+import play.api.data.Forms.{of, single}
+import play.api.data.format.Formatter
+import play.api.data.{Form, FormError}
 
 object CaptureLastMonthSubmittedForm {
 
-  val form: Form[Boolean] = {
-    Form(
-      "value" -> boolean
-    )
+  def formatter: Formatter[String] = new Formatter[String] {
+    override def bind(key: String, data: Map[String, String]): Either[Seq[FormError], String] = {
+      data.get(key) match {
+        case Some(x) => Right(x)
+        case _ => Left(Seq(FormError(key, "capture-last-month-submitted.error.message")))
+      }
+    }
+
+    override def unbind(key: String, value: String): Map[String, String] = Map(key -> value)
+  }
+
+  val form: Form[String] = {
+    Form(single(
+      "return_date" -> of(formatter)
+    ))
```

Comment:
I'm not sure it needs to be this complicated, you could probably use text.verifying and a constraint like this https://github.com/hmrc/vat-sign-up-frontend/blob/master/app/uk/gov/hmrc/vatsignupfrontend/forms/CompanyUtrForm.scala

Reply:
or you could use the month form from vat sign up